x/twap/api.go

```diff
@@ -97,7 +119,7 @@ func (k Keeper) getTwap(
 		return sdk.Dec{}, err
 	}
 
-	return strategy.computeTwap(startRecord, endRecord, quoteAssetDenom)
+	return strategy.computeTwap(startRecord, endRecord, quoteAssetDenom), err
```

Comment:
We should call `computeTwap` function here, passing in a `strategy` as an argument